modules/porous_flow/test/tests/jacobian/tests

```diff
@@ -162,6 +162,18 @@
     ratio_tol = 1E-7
     difference_tol = 1E10
   [../]
+  [./eff_stress03]
+    type = 'PetscJacobianTester'
+    input = 'eff_stress03.i'
+    ratio_tol = 1E-7
+    difference_tol = 1E10
```

Comment:
Start restricting these tests if you know they won't work in parallel or with threads: `max_parallel = 1, max_threads=1`